icu4j/main/core/src/main/java/com/ibm/icu/text/UnicodeSet.java

```diff
@@ -278,14 +283,25 @@
  *     </tr>
  *   </table>
  * </blockquote>
- * <p>To iterate over contents of UnicodeSet, the following are available:
- * <ul><li>{@link #ranges()} to iterate through the ranges</li>
- * <li>{@link #strings()} to iterate through the strings</li>
- * <li>{@link #iterator()} to iterate through the entire contents in a single loop.
- * That method is, however, not particularly efficient, since it "boxes" each code point into a String.
+ *
+ * <p>To iterate over contents of {@code UnicodeSet}, the following are available:
+ * <ul>
+ *   <li>to iterate through the ranges: {@link #ranges()}, {@link UnicodeSetIterator#nextRange()},
```

Comment:
minor: I would move the UnicodeSetIterator versions to the ends of each list item